app/views/idv/in_person/ready_to_verify/show.html.erb

```diff
@@ -232,24 +232,24 @@
 <p><%= t('in_person_proofing.body.expect.info') %></p>
 
 <p class="margin-top-3 margin-bottom-4">
-  <% if @presenter.service_provider_homepage_url.present? %>
-    <%= render ClickObserverComponent.new(event_name: 'IdV: user clicked sp link on ready to verify page') do %>
-      <%= t(
-            'in_person_proofing.body.barcode.return_to_partner_html',
-            link_html: link_to(
-              t(
-                'in_person_proofing.body.barcode.return_to_partner_link',
-                sp_name: @presenter.sp_name,
-              ),
-              @presenter.service_provider_homepage_url,
-            ),
-          ) %>
-    <% end %>
-  <% else %>
+  <% if @presenter.service_provider_homepage_url.blank? %>  
     <%= t('in_person_proofing.body.barcode.close_window') %>
   <% end %>
 </p>
 
 <%= render PageFooterComponent.new do %>
+  <% if @presenter.service_provider_homepage_url.present? %>
+  <%= render ClickObserverComponent.new(event_name: 'IdV: user clicked sp link on ready to verify page') do %>
+    <%= link_to(
+          t(
+            'in_person_proofing.body.barcode.return_to_partner_link',
+            sp_name: @presenter.sp_name,
+          ),
+          @presenter.service_provider_homepage_url,
+          class: 'display-inline-block padding-bottom-2',
+        ) %>
+          <% end %>
+          <% end %>
+          <br />
```

Comment:
Should the line break only be present if the SP homepage link is rendered? i.e. within the `if` block

Nit: Suggest to fix up the indentation here for readability.

```suggestion
  <% if @presenter.service_provider_homepage_url.present? %>
    <%= render ClickObserverComponent.new(event_name: 'IdV: user clicked sp link on ready to verify page') do %>
      <%= link_to(
            t(
              'in_person_proofing.body.barcode.return_to_partner_link',
              sp_name: @presenter.sp_name,
            ),
            @presenter.service_provider_homepage_url,
            class: 'display-inline-block padding-bottom-2',
          ) %>
    <% end %>
    <br />
  <% end %>
```